docs/source/packages/accounts-2fa.md

```diff
@@ -89,47 +95,35 @@ To call this function the user must be already logged in.
 
 Now that you have a way to allow your users to enable 2FA on their accounts, you can create a login flow based on that.
 
-To verify whether or not a user has 2FA enabled, you can call the function `Accounts.has2faEnabled`:
-
-{% apibox "Accounts.has2faEnabled" "module":"accounts-base" %}
-
 As said at the beginning of this guide, this package is currently working with two other packages: `accounts-password` and `accounts-passwordless`. Below there is an explanation on how to use this package with them.
 
 <h3 id="working-with-accounts-password">Working with accounts-password</h3>
 
-With the function `Accounts.has2faEnabled`, you can check whether or not the user has 2FA enabled, and based on this information, you can directly call `Meteor.loginWithPassword` if the 2FA is not enabled, or redirect the user to a place where they can provide a code, in case they do have 2FA enabled.
+When calling the function `Meteor.loginWithPassword`, if the 2FA is enabled for the user, an error will be return to the callback, so you can redirect the user to a place where they can provide a code.
 
-A way of using it would be:
+As an example:
 
 ```js
 <button 
   onClick={() => {
-    Accounts.has2faEnabled(username, (err, isEnabled) => {
-      if (err) {
-        console.error("Error verifying if user has 2fa enabled", err);
-        return;
-      }
-
-      if (isEnabled) {
-        // send user to a page or show a component 
-        // where they can provide a 2FA code
-        setShouldAskCode(true);
-        return;
-      }
-      // Normal login when they don't have 2FA enabled.
-      Meteor.loginWithPassword(username, password, error => {
-        if (error) {
-          console.error("Error trying to log in (user without 2fa)", error);
+    Meteor.loginWithPassword(username, password, error => {
+      if (error) {
+        if (error.reason === '2FA code must be informed') {
```

Comment:
It would be better to check a code or an enum here instead of the string

Reply:
I agree.

Reply:
:+1: